### PR TITLE
feat: add function to call api file size endpoint after s3 upload

### PR DESCRIPTION
**Pull request by jherr-dev**
Updates the add input file response with a file id and then calls a new api endpoint to set the file size in the DB after uploading the file to s3.

### CODE REVIEW

**Review by lebovic — APPROVED** [review]
LGTM

File: toolchest_client/api/query.py [hunk 1]
```diff
@@ -150,8 +150,25 @@ def _send_initial_request(self, tool_name, tool_version, tool_args,
 
         return create_response
 
+    def _update_file_size(self, fileId):
+        update_file_size_url = "/".join([
+            PIPELINE_URL,
```
[the remark this answers is not in the document]
Comment (bcai2) [reply toolchest_client/api/query.py:155]: I don't recall. I'd be fine refactoring this into `PIPELINE_SEGMENT_INSTANCES_URL` and `PIPELINE_SEGMENT_INSTACES_ROUTE` in `api/urls.py` and elsewhere.

[the remark this answers is not in the document]
Comment (bcai2) [reply toolchest_client/api/query.py:155]: @jherr-dev I can add a quick refactor commit to this branch/PR, unless you want to take the reins.

**Review by bcai2 — APPROVED** [review]
LGTM, though I am concerned about exposing _update_file_size() to the client, especially if it's used in things like tracking tool usage and billing. It would be best if the file size verification + update was done entirely on the API side after a completed upload.

**Comment by bcai2** [comment]
@jherr-dev  One last thing if you do decide to merge as-is: can you bump the version number of the package (in `pyproject.toml`) before merging this branch and future features/fixes? 
(We'll probably want to add a version number bump via CircleCI at some point.)

File: toolchest_client/api/query.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (bcai2) [reply toolchest_client/api/query.py:155]: Actually, upon re-reading the code, there are a few things to clean up around this part of the code -- I'll just open up a new chore PR.

**Pull request merged**